src/Geno/C++/Components/Project.h

```diff
@@ -26,6 +26,14 @@
 
 class ICompiler;
 
+struct FileFilter
+{
+	std::filesystem::path                m_Name;
+	std::filesystem::path                m_Path;
+	std::vector< std::filesystem::path > m_Files;
```

Comment:
Structs should not have their members prefixes with `m_` actually!

Reply:
Aight, noted